pyproject.toml

```diff
@@ -1,7 +1,55 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+# TODO: make sure files in MANIFEST.in are included
+[project]
+version = "0.0.0"
+name = "trio-monitor"
+description = "A monitor utility for Trio"
+readme = "README.rst"
+license = "MIT OR Apache-2.0"
+dependencies = ["trio"]
+requires-python = ">=3.5"
+authors = [
+	{ name = "Laura F. Dickinson", email = "l@veriny.tf" }
```

Comment:
At least the email here should be changed (at least whois doesn't work on the domain), maybe more?

Reply:
You can just set it to your name if you want. I don't plan on committing things. Or my modern psuedonym + github email if you really wanna keep me on there.